projects/packages/my-jetpack/_inc/components/plans-section/style.module.scss

```
@@ -8,10 +8,9 @@
 	margin-bottom: calc( var(--spacing-base ) * 3 );
 }
 
-.external-link {
-	font-size: var( --font-body );
-	&:hover {
-		color: var( --jp-black );
-		text-decoration-thickness: var( --jp-underline-thickness );
+.actions-list {
+	li {
+		line-height: 24px;
+		margin: 0 0 8px 0;
```

Something that we should consider implementing via an RNA component. Maybe not, though 😅 .
Let's compute the margin based on the spacing base.
```suggestion
		margin: 0 0 alc( var(--spacing-base ) * 3 ) 0;
```